solutions/token-gated-content/components/Info.tsx

```diff
@@ -0,0 +1,107 @@
+import { useUserState } from '../hooks/useUser'
+
+import { CheckIcon } from '@heroicons/react/outline'
+import { Text, Link, Button, LoadingDots } from '@vercel/examples-ui'
+
+export const features = [
+  'Exclusive merchandise content',
+  'Events and conferences',
+  'First access to new feature releases',
+]
+
+export const Info = () => {
+  const { handleDisconnect, loading, userState, handleConnect } = useUserState()
+
+  const handleLogin = () => {
+    handleConnect()
+  }
+
+  const noMetamask = userState === 'noMetamask'
+  const showConnect = userState !== 'noMetamask' && userState !== 'connected'
+  const showDisconnect = userState === 'connected'
+
+  return (
+    <section className="flex flex-col flex-2 w-full gap-4 text-white">
+      <div className="relative p-6 bg-dark-accents-0 bg-clip-padding backdrop-filter backdrop-blur-sm bg-opacity-10  rounded-2xl shadow-sm flex flex-col ">
+        <div className="flex-1">
+          <Text variant="h1" className="text-white">
+            Token Gated Content example
```

Comment:
@gbibeaul putting a large text comment here since I'm not sure we'll align on the changes just yet. If/when we do I can edit the files in the PR 😄 

> Token-Gated Content 
> 
> This example shows how you can create exclusive areas of your website using Next.js, Wagmi, and Ethers. For the example, connecting a metamask wallet will demonstrate how token-gated content works for token holders. 
> 
> Access 
> 
> Token-gating content requires people to have the token you own to see exclusive content on your website.
> 
> You can use token-gated content for:
> - Exclusive merchandise content
> - Events and conferences 
> - Communities and forums 
> - First access to new features 
> 
> Install metamask to view the live example